src/libs/analytics/types.ts

```diff
@@ -0,0 +1,64 @@
+import { AmplitudeEvent } from 'libs/amplitude/events'
+import { logEventAnalytics } from 'libs/analytics/logEventAnalytics'
+import { AnalyticsEvent } from 'libs/firebase/analytics/events'
+
+export type AnalyticsProvider = {
+  disableCollection: () => Promise<void> | void
+  enableCollection: () => Promise<void> | void
+  logScreenView: (screenName: string) => Promise<void> | void
+  logEvent: (
+    eventName: { firebase?: AnalyticsEvent; amplitude?: AmplitudeEvent },
+    params?: Record<string, unknown>
+  ) => Promise<void> | void
+} & typeof logEventAnalytics
+
+type BaseThematicHome = {
+  homeEntryId: string
+  from?: never
+  moduleId?: never
+  moduleListId?: never
+}
+
+type CategoryBlockThematicHome = BaseThematicHome & {
+  from: 'category_block'
+  moduleId: string
+  moduleListId: string
+}
+
+type HighlightThematicBlockThematicHome = BaseThematicHome & {
```

Comment:
je trouve ça bizarre d'étendre un type en écrasant tout les "sous types" sauf 1

```suggestion
type CategoryBlockThematicHome = {
  homeEntryId: string
  from: 'category_block'
  moduleId: string
  moduleListId: string
}

type HighlightThematicBlockThematicHome = {
  homeEntryId: string
```